### ISSUE TITLE
Use enums instead of strings in flatbuffers

### ISSUE THREAD

**Issue by ibc** [versatica/mediasoup#995]
Once flatbuffers effort is done we should replace strings with proper enums everywhere in the messages exchanged between Node/Rust and worker. Just it.

**Comment by jmillan** [comment]
You mean logs sent from worker, right?

**Comment by ibc** [comment]
> You mean logs sent from worker, right?

No. I mean that we should send `ICE_STATE_CONNECTED` enum instead of `"connected"` string in `icestatechange` notification from worker to Node/Rust. I mean that we should send `CODEC_MIME_TYPE_AUDIO_OPUS` instead of `"audio/opus"` string in the `mimeType` field of a codec in the `produce` request from Node/Rust to the worker. Etc etc.

**Comment by jmillan** [comment]
Oh yes, there are few string there that should be enums, yes.

**Comment by ibc** [comment]
@jmillan should you link this issue to your ongoing PR?

**Comment by jmillan** [comment]
> @jmillan should you link this issue to your ongoing PR?

We should note in the issue description, via TODO note, the missing fields.

**Comment by ibc** [comment]
This was already done.

**Comment by jmillan** [comment]
BTW, there may be some missing types in flatbuffers that are present, ie: in Rust.

For example the [MimeType](https://github.com/versatica/mediasoup/blob/v3/rust/src/rtp_parameters.rs#L237).  I already made an attempt to use a type in the worker and failed. I leave this comment here in case someone wants to give it a try anytime.
